### DISCUSSION TITLE
Allow an option to disable message header metadata

### DISCUSSION THREAD

**Discussion by devtoor** [simple-login/app#1422] [Feature Requests]
![simplelogin](https://user-images.githubusercontent.com/32439792/201507086-fd03d770-b52c-4e35-b067-428c4a893e9c.png)

It is not a fully "anonymous email service" if the alias email is traceable back to you. Some aliases only exist for login purposes, and we don't respond to automated alerts/notifications/marketing emails. A simple "From: <reverse_alias> To: <personal_email>" is way better and add layer of privacy. The user's email is only anonymous from the website perspective, but not from the email service provider.

![email](https://user-images.githubusercontent.com/32439792/201507098-d296cfbb-73be-41e1-aa00-eb45fea3f62e.png)

![meta](https://user-images.githubusercontent.com/32439792/201507107-16af41fb-6970-4de2-ad3a-f6318f07a22d.png)

Adding the alias email to the encrypted body is a great idea. But adding the alias to the message header is a way to expose yourself using the login alias to the email provider and SMTP servers the email has to go through.  The email metadata can be used for correlation.

https://github.com/simple-login/app/blob/master/email_handler.py#L874

The alias is added inside the email body. The `X-Simplelogin-Envelope-To` can be remove.

```python
msg = add_header(
                msg,
                f"""Forwarded by SimpleLogin to {alias.email} with "{orig_subject}" as subject""",
                f"""Forwarded by SimpleLogin to {alias.email} with <b>{orig_subject}</b> as subject""",
            )
```

https://github.com/simple-login/app/blob/master/email_handler.py#L896

The `To` can be replaced with user email instead. Since only the body is encrypted using PGP, it would be nice if we can include the original-from email and the alias name in the mail body as well.

**Comment by devtoor** [comment]
```html
Forwarded by SimpleLogin
from: <b>{envelope.mail_from}</b>
to: {alias.name} <b>{alias.email}</b>
subject: <b>{orig_subject}</b>
------------------------------
```

**Reply by librafrog** [reply]
Maybe is a good thing they indeed make it a little more Tight and confidential. peace ✌🏼
